docker/nonvoting_mixnet/docker-compose.yml

```diff
@@ -85,6 +90,7 @@ services:
     user: $DOCKER_USER
     volumes:
       - ./conf/auth:/conf
+      - /opt/sphincsplus:/opt/sphincsplus
```

Comment:
`/opt/sphincsplus` and `/usr/local/sphincsplus` etc -- I thought we ditched `/opt` ? Either way, why does it need to be mounted as a docker volume? Shouldn't we just build it inside the docker build ?

Reply:
fixed